terraform/environments/data-platform/tests/delete_table.py

```diff
@@ -0,0 +1,38 @@
+import json
+import os
+import sys
+
+import boto3
+import requests
+
+
+data_product_name = "example_prison_data_product"
+table_name = "testing"
+base_url = "https://hsolkci589.execute-api.eu-west-2.amazonaws.com/development"
+schema_url = f"/data-product/{data_product_name}/table/{table_name}"
+url = base_url + schema_url
+glue = boto3.client("glue")
+
+try:
+    auth_token = json.loads(os.environ["API_AUTH"])
+    print(auth_token.keys())
```

Comment:
⚠️ We shouldn't be printing this, this could leak the credentials via github actions logs

Reply:
These tests are for running locally and aren't triggered by the workflow.